sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -301,12 +301,24 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
   def start(): StreamingQuery = startInternal(None)
 
   /**
+   * :: Experimental ::
```

Comment:
`Experimental` -> `Evolving`?

Reply:
Per the comment https://github.com/apache/spark/blob/cc23581e2645c91fa8d6e6c81dc87b4221718bb1/common/tags/src/main/java/org/apache/spark/annotation/Experimental.java#L28-L31 `:: Experimental ::` is the tag for scaladoc.

Reply:
Both annotations (experimental and evolving) provide different semantics, right? Adding the different tag would give more confusion as it's not clear whether this is experimental vs evolving. You're getting it from Experimental, not Evolving.

I'll reopen this.

Reply:
Make sense, let me delete this `experimental` one, only keep evolving.